src/main/java/me/itzg/helpers/modrinth/InstallModrinthModpackCommand.java

```diff
@@ -103,271 +68,55 @@ public class InstallModrinthModpackCommand implements Callable<Integer> {
     SharedFetchArgs sharedFetchArgs = new SharedFetchArgs();
 
     @Override
-    public Integer call() throws Exception {
-        final ModrinthModpackManifest prevManifest = Manifests.load(
-            outputDirectory, ModrinthModpackManifest.ID, ModrinthModpackManifest.class);
-
-        final ProjectRef projectRef;
-        final Matcher m = MODPACK_PAGE_URL.matcher(modpackProject);
-        if (m.matches()) {
-            final String versionName = m.group("versionName");
-            if (versionName != null && version != null) {
-                throw new InvalidParameterException("Cannot provide both project file URL and version");
-            }
-            projectRef = new ProjectRef(m.group("slug"), versionName != null ? versionName : version);
-        } else {
-            projectRef = new ProjectRef(modpackProject, version);
-        }
-
-        final ModrinthModpackManifest newManifest =
-            processModpack(projectRef, prevManifest);
-
-        if (newManifest != null) {
-            Manifests.cleanup(outputDirectory, prevManifest, newManifest, log);
-            Manifests.save(outputDirectory, ModrinthModpackManifest.ID, newManifest);
-        }
-
-        return ExitCode.OK;
-    }
-
-    /**
-     * @return the new manifest or null if already installed
-     */
-    private ModrinthModpackManifest processModpack(ProjectRef projectRef, ModrinthModpackManifest prevManifest) {
+    public Integer call() throws IOException {
+        final ModrinthApiClient apiClient = new ModrinthApiClient(
+            baseUrl, "install-modrinth-modpack", sharedFetchArgs.options());
 
-        try (ModrinthApiClient apiClient = new ModrinthApiClient(baseUrl, "install-modrinth-modpack", sharedFetchArgs.options())) {
-            return apiClient.getProject(projectRef.getIdOrSlug())
-                .onErrorMap(FailedRequestException::isNotFound,
-                    throwable ->
-                        new InvalidParameterException("Unable to locate requested project given " + projectRef, throwable)
-                )
-                .flatMap(project ->
-                    apiClient.resolveProjectVersion(
-                            project, projectRef, loader != null ? loader.asLoader() : null, gameVersion, defaultVersionType
-                        )
-                        .switchIfEmpty(Mono.defer(() -> Mono.error(new InvalidParameterException(
-                            "Unable to find version given " + projectRef)))
-                        )
-                        .doOnNext(version -> log.debug("Resolved version={} from projectRef={}", version.getVersionNumber(), projectRef))
-                        .publishOn(Schedulers.boundedElastic()) // since next item does I/O
-                        .filter(version -> needsInstall(prevManifest, project, version))
```

Comment:
@kMaiSmith I just realized you seem to have removed this "force synchronize" feature. Can you create a follow up PR to put it back?

Reply:
Created https://github.com/itzg/mc-image-helper/issues/290 to track this